crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -853,28 +854,35 @@ where
     let networking_fut = networking_fut;
     let farm_fut = farm_fut;
     let farmer_cache_worker_fut = farmer_cache_worker_fut;
+    let prometheus_worker = prometheus_worker;
 
     let networking_fut = pin!(networking_fut);
     let farm_fut = pin!(farm_fut);
     let farmer_cache_worker_fut = pin!(farmer_cache_worker_fut);
 
     select! {
         // Signal future
-        _ = signal.fuse() => {}
+        () = signal.fuse() => {}
 
         // Networking future
-        _ = networking_fut.fuse() => {
-            info!("Node runner exited.")
+        networking_error = networking_fut.fuse() => {
+            info!(?networking_error, "Node runner exited.")
         },
 
         // Farm future
         result = farm_fut.fuse() => {
+            info!("Farm exited.");
```

Comment:
More descriptive message (indicating which farm, with which result, etc.) is already logged on lines 817 and 820 above, this should not be necessary.